node/network/pov-distribution/src/lib.rs

```diff
@@ -482,13 +568,18 @@ async fn handle_incoming_pov(
 	).await
 }
 
+/// Handles a newly connected validator in the context of some relay leaf.
+fn handle_validator_connected(state: &mut State, peer_id: PeerId) {
+	state.peer_state.entry(peer_id).or_default();
```

Comment:
It doesn't seem to do anything differently than the regular `PeerConnected` logic?

Reply:
I've unified them, yes